### ISSUE TITLE
WCAG 2.2 development update: Tag || Target size (minimum)

### ISSUE THREAD

**Issue by davidc-gds** [alphagov/govuk-frontend#4062]
## What
Check whether the target area of tags needs to be adjusted so that the target area has a height of at least 24px.

Note: the typography work may affect this, as it is changing how font size is handled for small, mobile-size screens.

### Resources
- [Tag component](https://design-system.service.gov.uk/components/tag/)
- [WCAG 2.2 criterion](https://www.w3.org/TR/WCAG22/#target-size-minimum)

## Why
Currently tags on small, mobile-size viewport widths are rendered with a target area height of 23px. On large, desktop and tablet-size viewport widths, the height of the target area is 25px.

It would be better if we just bump small-screen up to be at least 24px height.

Notes on the scenario where this applies:

- The tag is made into a link or button (interactive element)
- The page with the tag is viewed on a mobile sized viewport width
- The tag is using our current version of the typography scale

The relevant part of the criteria is:

> "The size of the target for pointer inputs is at least 24 by 24 CSS pixels..."

Technically our current tag should be fine from a 'passes WCAG 2.2 AA' sense, in 99% of situations. However, the current form could fail AA on mobile-size screen widths, if a bunch of tags are stacked vertically without spacing between them.

## Who needs to work on this
Developer
## Who needs to review this
Accessibility specialist

## Done when

- [ ] Investigate the issue
- [ ] Determine whether the typography changes will fix this by bumping up the size of our mobile fonts
- [ ] Decide whether we see any value in adding a minimum width / height for the tag target area
- [ ] Make the change
- [ ] Review the change (accessibility specialist)
- [ ] Merge
